### ISSUE TITLE
Fail to remap key by following the tutorial

### ISSUE THREAD

**Issue by robot010** [dvdizon/kbd75-qmk-guide#3]
Hi, I tried to remap "CAPS LOCK" to "BACKSPACE" by using the http://qmkeyboard.cn/ website. I'm using KBF75 R6 and it just doesn't work. 

I already choose KFD75 at the website, and change the "caps" into "backspace" in the layout graph. Then switch to layer 1 so that those operations are confirmed. 

But it just doesn't work. Is there any suggestion on this issue?

**Comment by dvdizon** [comment]
Hey, thanks for stopping by.

Couple questions:
> Then switch to layer 1 so that those operations are confirmed.

What do you mean by this? I'm not quite following how switching to Layer 1 helps confirms operations?

> But it just doesn't work

To what extent is not working? 
* You are trying to flash the new layout and its not working?
* You successfully flashed the layout, but when you press the button it's not backspace for your CAPS LOCK?

As any suggestion in this issue: please feel free to drop by the mechanical keyboards discord (discord.gg/mechkeys) and ask in the #kb-help channel

**Comment by botao-deng** [comment]
Hi david, 

Thanks for your quick reply! I have solved the problems by following the "Getting Started" at QMK official website. 

I was having that naive thought that changing the layout at the  http://qmkeyboard.cn/ website could change the layout of my keyboard automatically. Therefore I was asking if there is any "confirm" option at the website once we customized the new layout. But now I am able to do it with the QMK toolbox. 

Thanks for your help and I'm looking forward to exploring more about the keyboard!
